kernel/src/arch/aarch64/board/raspi3/mod.rs

```diff
@@ -10,21 +10,24 @@ pub mod timer;
 
 use crate::drivers::gpu::fb::{self, ColorDepth, ColorFormat, FramebufferInfo, FramebufferResult};
 
+pub const BOARD_NAME: &'static str = "Raspberry Pi 3";
 pub const PERIPHERALS_START: usize = bcm2837::addr::PERIPHERALS_START;
 pub const PERIPHERALS_END: usize = bcm2837::addr::PERIPHERALS_END;
+pub const CPU_NUM: usize = 4;
+
+#[no_mangle]
+pub static CPU_SPIN_TABLE: [usize; CPU_NUM] = [0xd8, 0xe0, 0xe8, 0xf0];
```

Comment:
What's the meaning of these magic numbers?

Reply:
See https://github.com/torvalds/linux/blob/master/arch/arm/boot/dts/bcm2837.dtsi#L46 and https://github.com/qemu/qemu/blob/master/hw/arm/raspi.c#L67. Secondary CPUs will spin until the spin table holds a jump address.

Reply:
Got it